lib.php

```diff
@@ -335,7 +335,25 @@ function local_annoto_extend_settings_navigation(settings_navigation $settingsna
 
     // Check and create LTI external tool.
     require_once($CFG->dirroot . '/mod/lti/locallib.php');
-    $lti = lti_get_tool_by_url_match($settings->toolurl);
+
+    // Will return all available tools by domain from toolurl.
+    $possibletools = lti_get_tools_by_domain($settings->customdomain, LTI_TOOL_STATE_CONFIGURED, null);
```

Comment:
for production customdomain will not be used and be empty instead preset domain is used, please fix according to previous similar domain resolution and our discussion